launchable/commands/record/session.py

```diff
@@ -73,13 +81,24 @@ def session(
     you should set print_session = False because users don't expect to print session ID to the subset output.
     """
 
+    if not is_no_build and not build_name:
+        raise click.UsageError("Error: Missing option '--build'")
+
+    if is_no_build and (read_build() and read_build() != ""):
+        raise click.UsageError(
+            'The cli already created .launchable file. If you want to use `--no-build` option. Please remove `.launchable` file before executing.')  # noqa: E501
```

Comment:
`'The cli already created .launchable file. If you want to use `--no-build` option, please remove `.launchable` file before executing.'`